src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState, memo } from 'react';
+import ActionsArrowDownIcon from 'icons/react/ActionsArrowDownIcon';
+import ActionsArrowUpIcon from 'icons/react/ActionsArrowUpIcon';
+import { IconButton } from 'components';
+import HamburgerBarsIcon from 'icons/react/HamburgerBarsIcon';
+import DenyIcon from 'icons/react/DenyIcon';
+import CBILogo from './CBI_Logo_White.svg';
+import {
+  WhoWeServeDropdown,
+  HowWeHelpDropdown,
+  WhatWeOfferDropdown,
+  ResourcesDropdown,
+} from './constants';
+import NavBarDropdown from './NavbarDropdown';
+
+const NAV_LIST = 'navList';
+const WHO_WE_SERVE = 'whoWeServe';
+const HOW_WE_HELP = 'howWeHelp';
+const WHAT_WE_OFFER = 'whatWeOffer';
+const RESOURCES = 'resources';
+
+const DEFAULT_STATE: { [key: string]: boolean } = {
```

Comment:
what do you think about something like this?

`type Section = 'navList' | 'whoWeServe' | 'howWeHelp' | 'whatWeOffer' | 'resources';`

`const DEFAULT_STATE: Record<Section, boolean> = {`

`const handleClick = (sectionName: Section): void => {
    setShowClass({ ...showClass, [sectionName]: !showClass[sectionName] });
  };

  const getArrowUpOrDown = (isOpen: boolean, sectionName: Section): JSX.Element => (`

in this case we have a strong type and we will throw an error if someone is trying to add a new key that it is not on the Section type